pkg/sql/logictest/testdata/logic_test/crdb_internal

```diff
@@ -228,17 +228,28 @@ node_id  component  field   value
 1        UI         URI     /
 
 query ITTTTT colnames
-SELECT node_id, network, regexp_replace(address, '\d+$', '<port>') as address, attrs, locality, regexp_replace(version, '^\d+\.\d+(-\d+)?$', '<version>') as version FROM crdb_internal.gossip_nodes WHERE node_id = 1
+SELECT node_id, network, regexp_replace(address, '\d+$', '<port>') as address, attrs, locality, regexp_replace(server_version, '^\d+\.\d+(-\d+)?$', '<server_version>') as server_version FROM crdb_internal.gossip_nodes WHERE node_id = 1
 ----
-node_id  network  address           attrs  locality  version
-1        tcp      127.0.0.1:<port>  []     ·         <version>
+node_id  network  address           attrs  locality  server_version
+1        tcp      127.0.0.1:<port>  []     {}        <server_version>
 
 query IITBB colnames
 SELECT node_id, epoch, regexp_replace(expiration, '^\d+\.\d+,\d+$', '<timestamp>') as expiration, draining, decommissioning FROM crdb_internal.gossip_liveness WHERE node_id = 1
 ----
 node_id  epoch  expiration    draining  decommissioning
 1        1      <timestamp>   false     false
 
```

Comment:
Could you add queries here and below that retrieve things like node_id, store_id, attrs, locality? Those should be static and so we'll see at least some data.

Reply:
Sure, implemented.